README.md

```diff
@@ -141,6 +141,16 @@ optimizeit --help
 optimizeit -h
 ```
 
+### Token Usage
+
+This flag allows you print the completion token, prompt token, and total token that OptimizeIt is using from your prompt, usage:
+
+```bash
+optimizeit file1 --token-usage
+optimizeit file1 file2 -tu
```

Comment:
NOTE: you typically want to use a single arg for `-` vs. `--` when doing flags.  I'd suggest `-u`.  A few CLIs break this rule, but most Unix tools follow it.

Reply:
Oh interesting! I didn't know that, thanks for pointing it out